src/executor/executorManager.ts

```diff
@@ -790,8 +790,21 @@ export class ExecutorManager {
         await this.refreshUserOperationStatuses()
 
         // for all still not included check if needs to be replaced (based on gas price)
-        const gasPriceParameters =
-            await this.gasPriceManager.tryGetNetworkGasPrice()
+        let gasPriceParameters: {
+            maxFeePerGas: bigint
+            maxPriorityFeePerGas: bigint
+        }
+
+        try {
+            gasPriceParameters =
+                await this.gasPriceManager.tryGetNetworkGasPrice()
+        } catch {
```

Comment:
should we set `this.currentlyHandlingBlock = false` here when it early exits due to not finding gasPrice